lib/elixir/test/elixir/code_formatter/operators_test.exs

```diff
@@ -8,6 +8,39 @@ defmodule Code.Formatter.OperatorsTest do
   @short_length [line_length: 10]
   @medium_length [line_length: 20]
 
+  describe "nullary" do
+    test "formats symbol operators" do
+      assert_same ".."
+    end
+
+    test "combines with unary and binary operators" do
+      assert_same "not .."
+      assert_same "left = .."
+      assert_same ".. = right"
+    end
+
+    test "is wrapped in parentheses on ambiguous calls" do
+      assert_same "require (..)"
+      assert_same "require foo, (..)"
+      assert_same "require (..), bar"
+      assert_same "require(..)"
+      assert_same "require(foo, ..)"
+      assert_same "require(.., bar)"
+
+      assert_same "assert [.., :ok]"
```

Comment:
```suggestion
      assert_same "assert [.., :ok]"
      assert_same "assert {.., :ok}"
```

Perhaps this is a good sanity test as well?

Reply:
I mean, I assume the grammar context would be the same (or almost the same) for (), [], {} and <<>>